### ISSUE TITLE
generation results are different between "diffusers", "huggingface gradio" and "modelscope gradio"

### ISSUE THREAD

**Issue by MengHao666** [Tongyi-MAI/Z-Image#9]
prompt:
"一位东亚女性，年龄约25-35岁，面容精致，肤色白皙，眉形修长，眼妆强调眼线与浓密睫毛，腮红自然，唇部涂有饱满正红色口红，整体妆容明艳而典雅。她身着墨绿色丝绒材质的修身改良旗袍，领口与袖口饰有金色滚边，盘扣为同色系，凸显华贵质感。发色深棕，梳成复古手推波浪卷短发，耳垂佩戴一对翠绿通透的翡翠耳坠。她端庄地坐在一张梨花木椅上，右手轻托一杯清茶，杯中热气袅袅升腾。背景为中式茶馆一隅，身后是雕花木窗格与一幅悬挂的水墨山水画，窗外光线透过窗棂洒入，在室内形成柔和光影，整体氛围宁静、典雅而高贵。"

the following results are all with default settings, We could find results of  "diffusers" is too much more oill than the others.
What cause such big difference? Or do u use different model versions across different platform? What setting is best if I want get the real best results?

result1:  using the following "diffusers" inference code in your repo, I met the following result
code: 
```
import torch
from diffusers import ZImagePipeline

# 1. Load the pipeline
# Use bfloat16 for optimal performance on supported GPUs
pipe = ZImagePipeline.from_pretrained(
    "Tongyi-MAI/Z-Image-Turbo",
    torch_dtype=torch.bfloat16,
    low_cpu_mem_usage=False,
)
pipe.to("cuda")

# [Optional] Attention Backend
# Diffusers uses SDPA by default. Switch to Flash Attention for better efficiency if supported:
# pipe.transformer.set_attention_backend("flash")    # Enable Flash-Attention-2
# pipe.transformer.set_attention_backend("_flash_3") # Enable Flash-Attention-3

# [Optional] Model Compilation
# Compiling the DiT model accelerates inference, but the first run will take longer to compile.
# pipe.transformer.compile()

# [Optional] CPU Offloading
# Enable CPU offloading for memory-constrained devices.
# pipe.enable_model_cpu_offload()

prompt = "Young Chinese woman in red Hanfu, intricate embroidery. Impeccable makeup, red floral forehead pattern. Elaborate high bun, golden phoenix headdress, red flowers, beads. Holds round folding fan with lady, trees, bird. Neon lightning-bolt lamp (⚡️), bright yellow glow, above extended left palm. Soft-lit outdoor night background, silhouetted tiered pagoda (西安大雁塔), blurred colorful distant lights."

# 2. Generate Image
image = pipe(
    prompt=prompt,
    height=1024,
    width=1024,
    num_inference_steps=9,  # This actually results in 8 DiT forwards
    guidance_scale=0.0,     # Guidance should be 0 for the Turbo models
    generator=torch.Generator("cuda").manual_seed(42),
).images[0]

image.save("example.png")

```

result:
<img width="1328" height="1328" alt="Image" src="https://github.com/user-attachments/assets/2acad033-2f86-4fec-b299-ea9d6479c5ea" />

rsult 2: using huggingface gradio

<img width="1024" height="1024" alt="Image" src="https://github.com/user-attachments/assets/d0682182-48f6-421c-a8ec-308ac6070186" />

result3: modelscope gradio

![Image](https://github.com/user-attachments/assets/7132225d-7872-4e6d-bc06-e394482ee8a1)

**Comment by JerryWu-code** [comment]
Here is the image which I generate using newest version of diffusers with your setting but in "_flash_3"(left) & "_native_flash"(right) two different backends, same prompt, seed 42, in H800. 

<img width="1124" height="564" alt="Image" src="https://github.com/user-attachments/assets/ba1ab6b2-5099-4f2a-ab0e-a8268e2c19c1" />

And here is the result which I generated from huggingface demo, this may result a bit different with the previous two (coz huggingface demo we use "_flash_3_hub" backend along with "kernels" package supported by huggingface space and integrated their native AoIT(Ahead of Time) compilation. So the difference would emerge with difference **devices** deployment & **attention accelerate backend, kernels**.

<img width="761" height="758" alt="Image" src="https://github.com/user-attachments/assets/858e79e3-a1d7-4a7b-ade0-726bd5935056" />

But in all, the differences would be that big as you inplement when running locally, so could you double check the **prompt** that you use again, the **device** that you use, as well the **environment** (uninstall then install from latest diffusers source code).

**Comment by MengHao666** [comment]
thanks for your reply, I will retry immediatly now!
